libs/paypal/src/selectors/paypal.selector.ts

```diff
@@ -53,7 +53,7 @@ const createPaypalSelectors = <T extends DaffPaypalTokenResponse>(): DaffPaypalM
 
 export const daffPaypalSelectors = (() => {
 	let cache;
-	return <T extends DaffPaypalTokenResponse>(): DaffPaypalMemoizedSelectors<T> => cache = cache 
+	return <T extends DaffPaypalTokenResponse = DaffPaypalTokenResponse>(): DaffPaypalMemoizedSelectors<T> => cache = cache 
```

Comment:
I don't think this is necessary.

Reply:
why not?

Reply:
without the default value set, `daffPaypalSelectors()` still has the type `DaffPaypalMemoizedSelectors<DaffPaypalTokenResponse>`. Check to make sure thats what you see as well.

Reply:
hmmm, yea that's what I see too. Seems pretty weird though. Like maybe typescript automatically picks a default for function calls that are generic. I think it should probably stay here though, so it's clear that this will pick a default for you. I'd rather rely on something more explicit, rather than some obscure rule buried in typescript somewhere. What do you think?

Reply:
yeah I agree, its probably better to be explicit. You should make this change in product and category as well.

Reply:
It should be done in product and category already